src/problems/RadMarshakAsymptoticCorr/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+add_executable(test_radiation_marshak_asymptotic_corr ../RadMarshakAsymptotic/test_radiation_marshak_asymptotic.cpp ../../util/fextract.cpp ../../math/interpolate.cpp ${QuokkaObjSources})
+
+if(AMReX_GPU_BACKEND MATCHES "CUDA")
+    setup_target_for_cuda_compilation(test_radiation_marshak_asymptotic_corr)
+endif(AMReX_GPU_BACKEND MATCHES "CUDA")
+
+add_test(NAME MarshakWaveAsymptoticDiffusion COMMAND test_radiation_marshak_asymptotic_corr MarshakAsymptoticCorr.in ${QuokkaTestParams} WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/tests)
```

Comment:
Is there a reason to build this twice? You could just call `add_test` with the `test_radiation_marshak_asymptotic` executable.

Reply:
You could even have both `add_test` statements in the same `CMakeLists.txt` file.

Reply:
Good point. Done!